src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/GeneralPartnerController.java

```diff
@@ -43,6 +45,25 @@ public GeneralPartnerController(GeneralPartnerService generalPartnerService) {
         this.generalPartnerService = generalPartnerService;
     }
 
+    @GetMapping("/{" + URL_PARAM_GENERAL_PARTNER_ID + "}")
+    public ResponseEntity<GeneralPartnerDto> getGeneralPartner(@RequestAttribute(TRANSACTION_KEY) Transaction transaction,
+                                                               @PathVariable(URL_PARAM_GENERAL_PARTNER_ID) String submissionId,
```

Comment:
I would call this parameter `generalPartnerId` not `submissionId`.

Reply:
Renamed